crypto/asymmetric/asymmetric_old/asymmetric_old_test.go

```diff
@@ -0,0 +1,284 @@
+package asymmetric_old
```

Comment:
The `asymmetric_test.go` still lives unchanged, so the new implementation continues to verify the test we had.

In this new `asymetric_old_test.go`, I added a complete copy-paste of *all* the previous implementation. (not just the changed part, so we can be sure about complete compatibility).

This tests do some bi-directional tests between encryption/decryption using new and old, old and new implementations. The idea is to be sure that old and new world still works correctly toegheter.

Reply:
Nice!